src/mesh/nemesis_io_helper.C

```diff
@@ -2385,7 +2336,7 @@ void Nemesis_IO_Helper::write_nodal_solution(const std::vector<Number> & values,
 
       for (int i=0; i<num_nodes; ++i)
         {
-          Number value = values[this->exodus_node_num_to_libmesh[i]*num_vars + c];
+          Number value = values[(this->exodus_node_num_to_libmesh[i]-1)*num_vars + c];
```

Comment:
I haven't thought about this in detail, but my initial reaction is "he's subtracting 1 from a libmesh node id" which seems like it would be bad if the libmesh node id is 0?

Reply:
Sadly, it's not that straightforward.  Subtracting 1 from a libmesh-node-id-plus-one (which gets done because we want 1-based numbering in the file and we directly pass that vector::data() to an Exodus API.  See the "push_back(id+1)" call and preceding comment.

Should we rename that vector?  And move it while we're at it; I'm not sure why it's in exodusII_io_helper.h if it's only ever used by the subclass.

Reply:
> Should we rename that vector? And move it while we're at it; I'm not sure why it's in exodusII_io_helper.h if it's only ever used by the subclass.

I'd be in favor of both the renaming and moving to the Nemesis helper. Not sure why it was added to the Exodus helper... maybe it was used there at some point.